lib/systems/examples.nix

```
@@ -248,8 +248,7 @@ rec {
 
   # BSDs
 
-  # Deprecate
-  amd64-netbsd = x86_64-netbsd;
+  amd64-netbsd = lib.warn "The amd64-netbsd system example is deprecated.  Use x86_64-netbsd instead." x86_64-netbsd;
```

nittiest nitpick. Not a strong preference but this what we use in nixpkgs:

```suggestion
  amd64-netbsd = lib.warn "The amd64-netbsd system example is deprecated. Use x86_64-netbsd instead." x86_64-netbsd;
```